backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/search/predicate/impl/ElasticsearchSearchPredicateBuilderFactoryImpl.java

```diff
@@ -87,30 +88,30 @@ public void contribute(ElasticsearchSearchPredicateCollector collector,
 
 	@Override
 	public MatchPredicateBuilder<ElasticsearchSearchPredicateBuilder> match(String absoluteFieldPath) {
-		return scopeModel
-				.getSchemaNodeComponent( absoluteFieldPath, PREDICATE_BUILDER_FACTORY_RETRIEVAL_STRATEGY, DslConverter.DISABLED )
-				.createMatchPredicateBuilder( searchContext, absoluteFieldPath, scopeModel.getDslConverterHandler() );
+		ElasticsearchScopedIndexFieldComponent<ElasticsearchFieldPredicateBuilderFactory> fieldComponent = scopeModel.getSchemaNodeComponent(
+				absoluteFieldPath, PREDICATE_BUILDER_FACTORY_RETRIEVAL_STRATEGY, DslConverter.DISABLED );
```

Comment:
You'll keep in mind you should remove the "DslConverter" parameter from here?

Reply:
Sure, I will as soon I can within this issue.